enhancements/declarative-index-config.md

```diff
@@ -628,89 +597,77 @@ $ opm index add --bundles quay.io/operatorhubio/etcd:v0.9.0 --mode replaces --ta
 $ docker push docker.io/my-namespace/community-operators:latest
 $ opm index inspect --image=docker.io/my-namespace/community-operators:latest --output=json
 $ cat community-operators/etcd.json
-[
-    {
-        "schema": "olm.package",
-        "name": "etcd",
-        "defaultChannel": "singlenamespace-alpha",
-        "icon": {
-            "base64data":"iVBORw0KGgoAAAANSUhEUgAAA.....",
-            "mediatype":"image/png"
-        },
-        "channels": ["alpha", "singlenamespace-alpha"],
-        "description": "A message about etcd operator, a description of channels"
+{
+    "schema": "olm.package",
+    "name": "etcd",
+    "defaultChannel": "singlenamespace-alpha",
+    "icon": {
+        "base64data":"iVBORw0KGgoAAAANSUhEUgAAA.....",
+        "mediatype":"image/png"
```

Comment:
I noticed that the package spec defines the icon as "embedded or remote", but I only see an example for embedded. What's are the remote fields?

Reply:
I think we want the ability to allow for an icon hosted remotely (referenced via a url instead of the enitre `base64data` byte array. But I don't think we have that capability right now, so we can tackle that (make changes to this config) when we do have that.